splash/pages/__init__.py

```diff
@@ -36,9 +36,9 @@ class Config:
         extra = Extra.forbid
 
 
-class Page(NewPage):
-    uid: str
-    document_version: int
+class Page(NewPage, CreatedVersionedDocument):
+    pass
+
 
 class NumVersions(BaseModel):
-    number: int
\ No newline at end of file
+    number: int
```

Comment:
I would prefer models that only have to do with routing be in the router.

Reply:
The reason this is here is because this is a model for the response we give when we tell the client the number of versions of a doc.

Reply:
Right...but it is limited in use to just routing, so why not in pages_routes.py?

Reply:
Yeah that makes sense